extras/indexing/src/main/java/mvm/rya/sail/config/RyaSailFactory.java

```diff
@@ -55,91 +52,98 @@
 import mvm.rya.rdftriplestore.inference.InferenceEngineException;
 
 public class RyaSailFactory {
-    @Deprecated
+    private static final Logger LOG = Logger.getLogger(RyaSailFactory.class);
+
     public static Sail getInstance(final Configuration conf) throws AccumuloException,
         AccumuloSecurityException, RyaDAOException, InferenceEngineException {
         return getRyaSail(conf);
     }
 
-    private static Sail getRyaSail(final Configuration config) throws AccumuloException, AccumuloSecurityException, RyaDAOException, InferenceEngineException {
+    private static Sail getRyaSail(final Configuration config) throws InferenceEngineException, RyaDAOException, AccumuloException, AccumuloSecurityException {
         final RdfCloudTripleStore store = new RdfCloudTripleStore();
-        RyaDAO crdfdao = null;
-        RdfCloudTripleStoreConfiguration conf;
+        final RyaDAO dao;
+        final RdfCloudTripleStoreConfiguration conf;
```

Comment:
Having a variable named 'config' and one named 'conf' is really confusing. Is 'conf' an updated version of 'config'? How do they differ?